### ISSUE TITLE
Problems with cysts

### ISSUE THREAD

**Issue by wangchuanxia** [neheller/kits19#26]
Dear organizers,
Thanks for organizing the excellent challenge.

I found that some renal tumors and renal cysts seem very similar. I wonder if the tumor segmentation will be confused if the cyst is labeled as normal kidney?

**Comment by neheller** [comment]
Thanks for participating, @wangchuanxia!

This issue is part of what makes this a difficult and interesting problem. Differentiation between renal cysts and neoplasms is well-studied and can be done by radiologists in CT with high accuracy. For a classic discussion of this topic, I'd look at [Bosniak 1986](https://pubs.rsna.org/doi/abs/10.1148/radiology.158.1.3510019). 

It might make an interesting addition to the dataset to go back and annotate all cysts, but that was not our priority for this challenge. Thus, with finite resources we faced a tradeoff between labeling cysts and labeling more tumors, and we decided that more tumors would be preferable. Perhaps this will be revisited in future challenges.
